examples/standalone/outputs.tf

```diff
@@ -33,3 +33,153 @@ output "bigquery_non_confidential_table" {
     module.regional_deid_pipeline
   ]
 }
+
+output "blueprint_type" {
+  description = "Type of blueprint this module represents."
+  value       = module.secured_data_warehouse.blueprint_type
+}
+
+output "cmek_bigquery_crypto_key" {
+  description = "The Customer Managed Crypto Key for the BigQuery service."
+  value       = module.secured_data_warehouse.cmek_bigquery_crypto_key
+}
+
+output "cmek_confidential_bigquery_crypto_key" {
+  description = "The Customer Managed Crypto Key for the confidential BigQuery service."
+  value       = module.secured_data_warehouse.cmek_confidential_bigquery_crypto_key
+}
+
+output "cmek_data_ingestion_crypto_key" {
+  description = "The Customer Managed Crypto Key for the data ingestion crypto boundary."
+  value       = module.secured_data_warehouse.cmek_data_ingestion_crypto_key
+}
+
+output "cmek_reidentification_crypto_key" {
+  description = "The Customer Managed Crypto Key for the reidentification crypto boundary."
+  value       = module.secured_data_warehouse.cmek_reidentification_crypto_key
+}
+
+output "tek_wrapping_keyring" {
+  description = "The name of tek wrapping key"
+  value       = module.tek_wrapping_key.keyring
+}
+
+output "centralized_logging_bucket_name" {
+  description = "The name of the bucket created for storage logging."
+  value       = module.centralized_logging.bucket_name
+}
+
+output "confidential_data_dataflow_bucket_name" {
+  description = "The name of the bucket created for dataflow in the confidential data pipeline."
+  value       = module.secured_data_warehouse.confidential_data_dataflow_bucket_name
+}
+
+output "data_ingestion_bucket_name" {
+  description = "The name of the bucket created for the data ingestion pipeline."
+  value       = module.secured_data_warehouse.data_ingestion_dataflow_bucket_name
+}
+
+output "data_ingestion_dataflow_bucket_name" {
+  description = "The name of the bucket created for dataflow in the data ingestion pipeline."
+  value       = module.secured_data_warehouse.data_ingestion_dataflow_bucket_name
+}
+
+output "confidential_data_project_id" {
+  description = "The Project where the confidential datasets and tables are created."
```

Comment:
no need to cap "project"

Reply:
might make the description consistent with the other project id's unless it's generally consistent with all the other examples that have this in their output.tf file.